matter_server/rootfs/etc/s6-overlay/s6-rc.d/matter-server/run

```diff
@@ -30,10 +30,17 @@ if ! bashio::var.has_value "${server_port}"; then
     extra_args+=('--listen-address' "$(bashio::addon.ip_address)")
 fi
 
-primary_interface="$(bashio::api.supervisor 'GET' '/network/info' '' 'first(.interfaces[] | select (.primary == true)) .interface')"
+primary_interface="$(bashio::api.supervisor 'GET' '/network/info' '' 'first(.interfaces[] | select (.primary == true)) .interface')";
+
+# Try fallback method (e.g. in case NetworkManager is not available)
+if [ -z ${primary_interface} ]; then
+  primary_interface="$(ip --json route show default | jq --raw-output '.[0].dev')";
```

Comment:
Let's add an info or warning message here, just so we know where the primary interface came from.

Reply:
Makes sense, I think warning is the appropriate level?
Because the intended mode of operation is that NetworkManger gives the correct primary interface